__tests__/parse-templates.test.ts

```diff
@@ -320,10 +328,11 @@ describe('parseTemplates', function () {
 
   it('hbs`Hello!` with imports alias', function () {
     const input =
-      "import { hbs as someHbs } from 'ember-cli-htmlbars';\n" +
+      "import someDefaultHbs, { hbs as someHbs } from 'ember-cli-htmlbars';\n" +
```

Comment:
Instead of modifying this test, can you add another test with an updated test title?

Something like:

> with default and named imports

Reply:
Makes sense. Done.